ecdf_vat_report/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'eCDF VAT Report',
+    'description': """
+        This modules allows to make VAT reports in XML format, ready for eCDF.
+    """,
+    'version': '10.0.0.0.1',
+    'licence': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'www.acsone.eu',
+    'depends': [
+        'l10n_lu_ext',
+        'mis_builder'
+    ],
+    'data': [
+        'views/ecdf_vat_report.xml',
+        'views/res_company.xml',
+
+        'data/mis_report.xml',
+        'data/mis_style.xml',
+        'data/mis_sub_kpi.xml',
+        'data/ecdf.manual.data.csv',
+        'data/mis.report.kpi.csv',
+        'data/mis.report.kpi.expression.csv',
+
+        'security/ir.model.access.csv',
+    ],
+    'demo': [
+        'demo/demo_ecdf_vat_report.xml',
+    ]
+}
```

Comment:
I agree, I'm changing the name.

Reply:
As we are looking to merge this module with l10n_lu_ecdf, I suggest to keep the name l10n_lu_ecdf for the final module. Does it make sens to you?

Reply:
l10n_lu_ecdf is fine with me, yes.

Reply:
The ecdf VAT report (as I implemented it) is a Model in opposition with the ecdf report (Profit & Loss, Balance Sheet and Chart of accounts) which is a TransientModel.
When you ask to merge both reports in the same module, your idea was to have ONE wizard to rule them all ^^ or keep them separated?
I prefer to keep the VAT report as a Model because we have the manual input that we may want to keep for history purpose.
What do you think?

Reply:
I'd be fine with one model to rule them all :) That could be a place to attach the resulting xml file.
For the P&L/BS, one can put several report in the xml file IIRC. I'm wondering if the annual VAT declaration can be included in the same XML as the other 3.

Reply:
Stephan,
Due to the fact that the fields to configure the VAT report are quite different from the other financial reports, I decided to do a little bit differently:
I create an AbstractModel called ecdf.abstract where all the common part are implemented.
the ecdf.vat.report and the ecdf.report (PL, CA & BS) inherit from that abstract model and of course I added the necessary fields and feature for both of them.
There are therefore two different menus "eCDF Financial Reports" and "eCDF VAT Reports" and two different views.
I decided to separate them because in the VAT Reports there are the manual inputs but we also want to visualize the MIS report before generating the XML.
Currently, in the financials reports, there's no way to preview the MIS report because you can generate more than one report (up to 3 in the same XML).

Do you agree with my approach or did I miss something?
If you think it is a good Idea, I can create the "feature", in the financial reports, to preview the different MIS report (One for PL, CA & BS). What do you think?